libs/community/langchain_community/utilities/sql_database.py

```diff
@@ -59,6 +59,7 @@ def __init__(
         view_support: bool = False,
         max_string_length: int = 300,
         lazy_table_reflection: bool = False,
+        custom_and_default_info: bool = False,
```

Comment:
Not sure if you're willing to tackle, but we'd benefit from general help improving documentation for this class (i.e., documenting the parameters + usage)

The class should provide functionality to retriever some raw information and allow the user to compose into an arbitrary prompt. It's OK that we have some default strategy for composition, but it's not flexible enough if users can't access the raw data